### ISSUE TITLE
[Trademark] Branding & Trademark Guidelines 

### ISSUE THREAD

**Issue by aeiras** [microprofile/microprofile-wg#58]
Refer to @manish2aug work with a few others via ticket https://github.com/eclipse/microprofile-marketing/issues/431

Final draft awaits Compatibility discussion.

**Comment by aeiras** [comment]
Hey Manish, @ @manish2aug, 

Can you please help with the trademark conversion into a google document? See Paul's latest request below.  Thank you, MicroProfiler!

https://github.com/microprofile/microprofile-wg/issues/93

**Comment by manish2aug** [comment]
Hi Amelia,

Sorry for the late reply, I created the Google doc from the Word doc and
merged some suggestions from Paul too.

There are still 2 things he mentioned I am not sure about

1. Question about multiple logos
2. Use of Steering committee instead of Working group

Thanks
Manish

**Comment by aeiras** [comment]
Manish, 

It seems that new document is no longer needed. Please see this ticket's exchange of today.  https://github.com/microprofile/microprofile-wg/issues/93

Can you withdraw new document from the drive and lets continue to work in the original document?  
Please pull me via the document with aerias@tomitribe.com for further questions on anything received you are unsure about.  
Thank you for leading this task so beautifully. 
You have nothing to apologize for!  :)

**Comment by aeiras** [comment]
Hey Manish, @manish2aug

do you want to own the last piece of adjustments via "suggestive mode" in the document that deal with adding that MP chose the Implementation Patent? 

Via this ticket or via the document, you are most welcomed to pull me in with Q&A. Thank you, Manish!

**Comment by manish2aug** [comment]
Hi Amelia, @aeiras 

Yes, I am up for it, just a bit out of context for some time. I Will go through the previous communications and would find out to ask relevant stuff if needed.

**Comment by aeiras** [comment]
Hey Manish, 

Welcome back to leading this task :) We adore you for it!

Now, there are 2 copies of the Guidelines, which is the one we are working on?  June 24th are the date for both documents: https://drive.google.com/drive/folders/1wbn8mUPTjQMpHYuzh1swwPCBnuS7rg3o

We must keep the one who has all the history of collaboration, OK.  You can move the unwanted copy to the Archives folder :) 

This Monday (10am PDT) we have the MPWG Trademark call.  You are most welcomed to ask questions via this ticket &/or attend that call and unblock the work, if necessary. 

Abrazos,
